pom.xml

```diff
@@ -368,6 +372,40 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>templatesIntegrationSmokeTests</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <properties>
+        <!-- Skip coverage checks, unit tests are skipped -->
+        <jacoco.skip>true</jacoco.skip>
+        <!-- Skip shade for faster runs -->
+        <skipShade>true</skipShade>
+        <!-- Some modules may yield no integration tests -->
+        <failIfNoTests>false</failIfNoTests>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>${surefire.version}</version>
+            <configuration combine.self="override">
+              <includes>
+                <include>**/*IT.java</include>
+              </includes>
+              <excludedGroups>
+                com.google.cloud.teleport.metadata.SkipDirectRunnerTest
+              </excludedGroups>
+              <groups>
+                ${direct-runner.tests}
+              </groups>
```

Comment:
These 2 could be contradictory. Maybe we simply don't annotate SkipDirectRunnerTest and keep a comprehensive list of DirectRunnerTest?

Reply:
I agree that would contradict, and in practice, no one is going to label a test with `@Category({DirectRunnerTest.class, SkipDirectRunnerTest.class})`, so it is essentially redundant too. I'll remove the `excludedGroups`